hardware:
  publish_rate: 100

joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50

head_trajectory_controller:
  type: position_controllers/JointTrajectoryController
  joints:
    - head_tilt_joint
    - head_pan_joint
  constraints:
    stopped_velocity_tolerance: 0.5
    goal_time: 0.05

arm_trajectory_controller:
  type: position_controllers/JointTrajectoryController
  joints:
    - arm_lift_joint
    - arm_flex_joint
    - arm_roll_joint
    - wrist_flex_joint
    - wrist_roll_joint
  constraints:
    stopped_velocity_tolerance: 0.5
    goal_time: 0.05

gripper_controller:
  type: position_controllers/JointTrajectoryController
  joints:
    - hand_motor_joint
  constraints:
    stopped_velocity_tolerance: 0.5
    goal_time: 0.05
